### PR TITLE
Added code to install ripgrep package for plugins

### PR DESCRIPTION
**Pull request by AltamashShaikh**
### Description:

Added code to install ripgrep package for plugins

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Comment by AltamashShaikh** [comment]
@sgiehl Is this the correct way to add this package globally ?

**Comment by sgiehl** [comment]
I guess that should work. Alternatively it could also be added to the prepare script here: https://github.com/matomo-org/github-action-tests/blob/main/scripts/bash/prepare.sh

**Comment by michalkleiner** [comment]
I also looked at the prepare script and it would be possible to add it there but it does not feel like the place where it would be better than in the action itself. If we had a script e.g. `install_dependencies.sh` or similar, that would be the best place, but we don't, so I think in the action it's ok.

**Comment by AltamashShaikh** [comment]
@sgiehl @michalkleiner Can we merge this ?
Created https://github.com/matomo-org/matomo/pull/22052 to remove it from core.

File: action.yml [hunk 1]
```diff
@@ -206,6 +206,12 @@ runs:
         WORKSPACE: ${{ github.workspace }}
         ACTION_PATH: ${{ github.action_path }}
 
+    - name: Install package ripgrep
+      if: inputs.plugin-name != ''
```
Comment (michalkleiner) [inline action.yml:210]: Is it always only for when a plugin is specified or should we install it always?

Comment (AltamashShaikh) [reply action.yml]: @michalkleiner Updated to install always, that would be better

**Comment by sgiehl** [comment]
I'll merge that now. We will see afterwards if it produces any problems in the test runs, but I don't think so.

**Pull request merged**